### PR TITLE
ocm/nucleus: add e2e

### PR DESCRIPTION
**Pull request by pmorie**
/wip

### CODE REVIEW

File: ci-operator/config/open-cluster-management/nucleus/open-cluster-management-nucleus-master.yaml [hunk 1]
```diff
@@ -48,6 +53,30 @@ tests:
...
     from: test-bin
+- as: e2e
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws
+    test:
+    - as: deploy
+      commands: |
+        KUBECTL=oc
+        export KUBECTL
```
Comment (qiujian16) [inline ci-operator/config/open-cluster-management/nucleus/open-cluster-management-nucleus-master.yaml:64]: i think with ocp, we also need to set OLM_NAMESPACE=openshift-operator-lifecycle-manager

Comment (pmorie) [reply ci-operator/config/open-cluster-management/nucleus/open-cluster-management-nucleus-master.yaml:64]: i believe you are correct, but i also believe that the last version i had pushed did not correctly provision a cluster for e2e.

**Pull request merged**